Source/ZoomNet/Models/CallLog.cs

```diff
@@ -0,0 +1,280 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// A call log item.
+	/// </summary>
+	public class CallLog
+	{
+		/// <summary>
+		/// Gets or sets the call Id.
+		/// </summary>
+		/// <value>The call id.</value>
+		[JsonPropertyName("id")]
+		public string Id { get; set; }
+
+		/// <summary>
+		/// Gets or sets the "accepted by" information.
+		/// </summary>
+		/// <value>Indicates who accepted the call.</value>
+		[JsonPropertyName("accepted_by")]
+		public CallLogTransferInfo AcceptedBy { get; set; }
+
+		/// <summary>
+		/// Gets or sets the call answer time.
+		/// </summary>
+		/// <value>
+		/// The call's answer time, in GMT date-time format.
+		/// The API only displays this response if the direction value is inbound.
+		/// </value>
+		[JsonPropertyName("answer_start_time")]
+		public DateTime AnswerStartTime { get; set; }
+
+		/// <summary>
+		/// Gets or sets the call end time.
+		/// </summary>
+		/// <value>The call end time, in GMT date-time format.</value>
+		[JsonPropertyName("call_end_time")]
+		public DateTime CallEndTime { get; set; }
+
+		/// <summary>
+		/// Gets or sets the unique id.
+		/// </summary>
+		/// <value>Unique identifier of the phone call.</value>
+		[JsonPropertyName("call_id")]
+		public string CallId { get; set; }
+
+		/// <summary>
+		/// Gets or sets the callee country code.
+		/// </summary>
+		/// <value>Country calling code.</value>
+		[JsonPropertyName("callee_country_code")]
+		public string CalleeCountryCode { get; set; }
```

Comment:
Here is the description of this property in Zoom Phone documentation:
`callee_country_code
string
Country calling code.`
Here is the example data returned with this property: "callee_country_code": "374",
Please see https://developers.zoom.us/docs/api/rest/reference/phone/methods/#operation/phoneUserCallLogs.

In your provided link, there is also such property:
`		/// <summary>Gets or sets the country code. For example, for United States phone numbers, this will be a +1 value.</summary>
		[JsonPropertyName("code")]
		public string CountryCode { get; set; }`

Reply:
Maybe you should simply change the property to nullable so the absence of a value in the JSON is converted to a null value in the C# property. 

```
public Country? MyCountry { get; set; }
```